docs/public/SUMMARY.md

```diff
@@ -0,0 +1,121 @@
+- [Installation](installation/README.md)
```

Comment:
Does `installation/README.md` exist?

In the generated mdbook, when I click on “Installation” in the outline (on the left hand side of the page), all I get is an empty page, which is rather awkward.

Reply:
It does not exist in the file structure.

Reply:
I'll remove the link in a follow-up PR.